manifests/config.pp

```diff
@@ -5,11 +5,26 @@
 #
 # @summary Manages Metricbeat's configuration file
 class metricbeat::config inherits metricbeat {
+
+  # check if array is empty, no need to create a config entry then
+  if $metricbeat::modules[0].length() > 0 {
+    $modules_arr = $metricbeat::modules
+  } else {
+    $modules_arr = undef
+  }
+
+  # if fields are "under root", then remove prefix
+  if $fields_under_root == true {
```

Comment:
Please refer to this variable as $metricbeat::fields_under_root. This is one of the causes for the build failures.